src/components/CopyModal.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import DialogContent from '@material-ui/core/DialogContent';
+import AssignmentIcon from '@material-ui/icons/Assignment';
+
+const CopyModal = ({ token, setOpenCopy }) => {
+  function copy() {
+    const copyText = document.getElementById('share-token');
+    copyText.select();
+    document.execCommand('copy');
+    setOpenCopy(false);
+  }
+
+  return (
+    <div>
+      <DialogContent>
+        <TextField
+          id="share-token"
+          autoFocus
```

Comment:
I would remove this, just because there's no reason to `autoFocus` this input. I wanna remove the other one the project as well. Seems like it can cause issues with visually impaired users. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#htmlattrdefautofocus

Reply:
Oh nice - thanks for the heads up!